docs/examples/viz_video_on_plane.py

```diff
@@ -0,0 +1,92 @@
+"""
+=======================================================
+Visualize Videos on Planes
+=======================================================
+
+The goal of this demo is to show how to visualize a video on
+Plane, which can be extended to potentially all shapes
```

Comment:
on a rectangle by updating a texture. Remove Plane, which can be extended to potentially all shapes. That should go to another tutorial (future work) no need to mention here.